src/dcm/agent/plugins/builtin/unmount_volume.py

```diff
@@ -17,24 +17,46 @@
 import dcm.agent.plugins.api.base as plugin_base
 import dcm.agent.plugins.api.utils as plugin_utils
 import dcm.agent.utils as utils
+import logging
 
+_g_logger = logging.getLogger(__name__)
+
+def _translate_ubuntu_device_id(deviceId):
+    if 'essd' in deviceId:
+        return deviceId
+    if deviceId.startswith("sd"):
```

Comment:
this should first check the existance of the device with something like:
if not os.path.exists(os.path.join("/dev/", deviceId)) and deviceId.startswith("sd"):